content/automate/ManageIQ/System/CommonMethods/MiqAe.class/__methods__/weightedupdatestatus.rb

```diff
@@ -29,6 +29,20 @@ def reconcile_states_percent(path, states)
               percent
             end
 
+            def create_cleanup_request(task)
+              sm_uri = @handle.object['cleanup_state_machine']
+              if sm_uri.present? && task.get_option(:cleanup_request_id).absent?
+                options = {}
+                sm_uri_array = sm_uri.split('/')
+                options[:instance_name] = sm_uri_array.pop
+                options[:class_name] = sm_uri_array.pop
+                options[:namespace] = sm_uri_array.join('/')
+                options[:user_id] = @handle.root['user_id']
+                request = @handle.execute("create_automation_request", options, 'admin', true)
+              end
+              task.set_option(:cleanup_request_id, request.id) unless request.nil?
```

Comment:
move this line inside the `if ... end` block

Reply:
Fixed.